installer.go

```diff
@@ -14,14 +14,16 @@ import (
 )
 
 type Installer struct {
-	manifest        *Manifest
-	appCacheDir     string
-	filesInAppCache map[string]interface{}
-	versionLine     *map[string]string
+	manifest                 *Manifest
+	appCacheDir              string
+	filesInAppCache          map[string]interface{}
+	versionLine              *map[string]string
+	retryTimeLimit           time.Duration
+	retryTimeInitialInterval time.Duration
 }
 
 func NewInstaller(manifest *Manifest) *Installer {
-	return &Installer{manifest, "", make(map[string]interface{}), &map[string]string{}}
+	return &Installer{manifest, "", make(map[string]interface{}), &map[string]string{}, 5 * time.Minute, 500 * time.Millisecond}
```

Comment:
I am wondering practically, if 5 minutes is too long for the retry window here. We've done 5 minutes in other code bases, but they exclusively run in our CI pipelines. We might want to shorten this down a bit possibly, so we have a slightly better user experience. Any thoughts?

Reply:
I liked 1 minute as a timeout (if it's giving errors for a minute, that's enough) and an initial retry after 1 second. I've gone with that now. You're right, 5 minutes was too long.